WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -297,6 +297,7 @@ - (UISwipeActionsConfiguration *)tableView:(UITableView *)tableView trailingSwip
     }
     
     UISwipeActionsConfiguration *swipeActions = [UISwipeActionsConfiguration configurationWithActions:actions];
+    swipeActions.performsFirstActionWithFullSwipe = FALSE;
```

Comment:
Sorry, one last request – in Objective-C, it's most common to write `NO` instead of `FALSE`. If you could update this, I'll get it merged in :)